internal/cli/ondemand_runner_inspect.go

```diff
@@ -0,0 +1,70 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/golang/protobuf/proto"
+
+	"github.com/hashicorp/waypoint-plugin-sdk/terminal"
+	"github.com/hashicorp/waypoint/internal/clierrors"
+	"github.com/hashicorp/waypoint/internal/pkg/flag"
+	pb "github.com/hashicorp/waypoint/internal/server/gen"
+	"github.com/posener/complete"
+)
+
+type OnDemandRunnerInspectCommand struct {
+	*baseCommand
+}
+
+func (c *OnDemandRunnerInspectCommand) Run(args []string) int {
+	// Initialize. If we fail, we just exit since Init handles the UI.
+	if err := c.Init(
+		WithArgs(args),
+		WithFlags(c.Flags()),
+		WithNoAutoServer(), // no auth in local mode
+		WithNoConfig(),
+	); err != nil {
+		return 1
+	}
+
+	if len(c.args) == 0 {
+		c.ui.Output("on-demand runner configuration ID required", terminal.WithErrorStyle())
+		return 1
+	}
+	id := c.args[0]
+
+	resp, err := c.project.Client().GetOnDemandRunnerConfig(c.Ctx, &pb.GetOnDemandRunnerConfigRequest{
+		Config: &pb.Ref_OnDemandRunnerConfig{Id: id},
+	})
+	if err != nil {
+		c.ui.Output(clierrors.Humanize(err), terminal.WithErrorStyle())
+		return 1
+	}
+	fmt.Println(proto.MarshalTextString(resp.Config))
```

Comment:
We've been trying to keep `inspect` commands looking the same, so I think this should output with the UI.NamedVars and a header before the config prints

Reply:
Can you show an example? I copied this from `auth-method inspect` haha. I'll copy whatever style.

Reply:
Ah, definitely!

I think initially we did it on `waypoint context inspect` and then it was suggested I keep that with https://github.com/hashicorp/waypoint/pull/2080.

Reply:
<img width="803" alt="CleanShot 2021-08-19 at 16 58 15@2x" src="https://user-images.githubusercontent.com/1299/130158748-7f90d9a0-519c-4061-866e-3088d8c43d58.png">

Done!